### PR TITLE
Added depositAndTransfer to VeilEther.

### PR DESCRIPTION
**Pull request by tomcbean**

### CODE REVIEW

**Comment by mertcelebi** [comment]
This looks good @tomcbean, thank you for the PR. I'll add a similar method to `VirtualAugurShares` as well. 

Unfortunately, the production version of `VeilEther` is already deployed, and it would be hard to swap. What's the use case for this change on your end?

File: contracts/VeilEther.sol [hunk 1]
```diff
@@ -51,6 +51,17 @@ contract VeilEther is UnlimitedAllowanceToken {
     return true;
   }
 
+  /**
+   * Buys tokens with Ether, exchanging them 1:1, and transfers them to a target address instead of msg.sender
+   *
+   * @param _target           Address to send the tokens
+   */
+  function depositAndTransfer(address _target) public payable returns (bool) {
```
Comment (mertcelebi) [inline contracts/VeilEther.sol:59]: This looks good. I know it's super simple, but do you mind adding tests before we merge?

**Review by mertcelebi — CHANGES_REQUESTED** [review]

**Comment by mertcelebi** [comment]
Closing this due to inactivity.

**Comment by tomcbean** [comment]
Sorry, I lost track of this. I thought it useful to include this additional function, but since the mainnet contract is already deployed, we can make due without. Thanks.

**Pull request closed without merging**